src/DotNet/ModuleDefMD.cs

```diff
@@ -1495,7 +1495,12 @@ public sealed class ModuleDefMD : ModuleDefMD2, IInstructionOperandResolver {
 			return new EmbeddedResourceMD(this, mr, Array2.Empty<byte>());
 		}
 
-		[HandleProcessCorruptedStateExceptions, SecurityCritical]	// Req'd on .NET Framework 4.0
+		// Required attributes on .NET Framework 4.0
+		// HandleProcessCorruptedStateExceptions is obsolete on .NET Core and newer
+#if !NETCOREAPP
+		[HandleProcessCorruptedStateExceptions]
+#endif
+		[SecurityCritical]
```

Comment:
I've gone ahead and used conditional compilation here to exclude the `HandleProcessCorruptedStateExceptions` attribute as it is marked with `Obsolete` and the obsolescence message claims this attribute is ignored by the runtime. I have not verified this behavior. An alternative could be to use `#pragma` directive to disable the obsolescence warning and still include the attribute in the compiled code just in case it does actually make a difference.